fiBeneficiary_test.go

```diff
@@ -33,7 +33,7 @@ func TestBlankFieldsNotIncluded(t *testing.T) {
 	fib.FIToFI.LineFour = ""
 	fib.FIToFI.LineSix = ""
 
-	require.Equal(t, "{6400}Line One*Line Three*Line Five*", fib.String())
+	require.Equal(t, "{6400}Line One**Line Three**Line Five*", fib.String())
```

Comment:
this seems that delimiter is being added for empty lines

Reply:
Yeah, true. But that is consistent with the rest of the tags. And for us it doesn't matter as the empty lines are all at the end. And perhaps if we have a case for a skipped line, this will preserve it which is more accurate.